### ISSUE TITLE
Add freya support

### ISSUE THREAD

**Issue by gonzalezb** [AldousP/MyDearWatson#4]
With freya RC being released soon please update script to 14.04 with freya updated packages.

**Comment by kenschnall** [comment]
Freya was released today!

**Comment by MAkerboom** [comment]
I like to try the new elementary too... :+1:

**Comment by theblacklion** [comment]
:chocolate_bar:

**Comment by aventu90** [comment]
Here is a modified script for Freya: http://www.reddit.com/r/Crouton/comments/32akho/any_guides_to_getting_the_new_elementary_freya/
I will try it later myself.

**Comment by DimitriDokuchaev** [comment]
That script does work, i already told skitals to (if he wants obviously, and if allowed by aldousP) to fork aldousP repo, make the changes and finally make a pull request.

**Comment by aventu90** [comment]
crouton doesn't unmount for me and the script breaks.. looks like an other issue

**Comment by DimitriDokuchaev** [comment]
yes I started having problems on unmounting statefull, but I don't know why, they stopped. and it now unmounts correctly go figure.

**Comment by bahaaaldin214** [comment]
juno is coming out is loki supported yet?
